canonical/jaas-dashboard
# Display a message when there are no machines

## Done

- Display a message when there are no machines in a model.
- Reenable Model tests and update to TypeScript and RTL.

## QA

- Click on a model that doesn't have any machines and then go to the 'Machines' tab.
- You should see a message and a link to the docs.

## Details

Fixes: #922.

## Screenshots

<img width="933" alt="Screen Shot 2022-10-25 at 2 23 55 pm" src="https://user-images.githubusercontent.com/361637/197675273-8fa51485-d125-4b78-ab84-4e6de3b3eeff.png">
